packages/flutter/test/widgets/gesture_disambiguation.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Files containing tests need to end in `_test.dart` to be picked up automatically by the test runner.

Reply:
done